servlet/6.0/_index.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Jakarta Servlet 6.0 (under development)"
+date: 2021-10-15
+summary: "Release for Jakarta EE 10"
+---
+Jakarta Servlet defines a server-side API for handling HTTP requests and responses.
+
+* [Jakarta Servlet 6.0 Release Record](https://projects.eclipse.org/projects/ee4j.servlet/releases/6.0)
+  * [Jakarta EE Platform 9 Release Plan](https://eclipse-ee4j.github.io/jakartaee-platform/jakartaee9/JakartaEE9ReleasePlan)
```

Comment:
This line should be removed (or replaced with a reference to the Jakarta EE 10 release plan, but that doesn't exist yet...)  :-)